app/konnect-platform/self-managed-migration.md

```diff
@@ -0,0 +1,269 @@
+To remove the `_workspace` key from the configuration, you can use the `yq` tool:
+
+```sh
+yq -i 'del(._workspace)' default.yaml
+yq -i 'del(._workspace)' inventory.yaml
+yq -i 'del(._workspace)' sales.yaml 
+```
+
+Synchronize the configuration to the Control Planes using decK configured with the 
+proper Control Plane name and the {{site.konnect_short_name}} access token:
+
+```sh
+deck gateway sync --konnect-token "$KONNECT_PAT" --konnect-control-plane-name default default.yaml
+deck gateway sync --konnect-token "$KONNECT_PAT" --konnect-control-plane-name inventory inventory.yaml
+deck gateway sync --konnect-token "$KONNECT_PAT" --konnect-control-plane-name sales sales.yaml
+```
+
+In this example, replace `$KONNECT_PAT` with your {{site.konnect_short_name}} PAT or specify your PAT as an environment variable.
+
+In addition to decK, {{site.konnect_short_name}} provides
+other tools that could be used for migrating your configuration. Each tool requires a different process. See their documentation for more information:
+
+* [Konnect Control Planes Config API](/api/konnect/control-planes-config/v2/#/)
+* [{{site.konnect_short_name}} Terraform Provider](/terraform/)
+
+### Data planes
+
+The recommended approach for migrating your Data Plane instances to {{site.konnect_short_name}} is to
+create new Data Plane instances connected to your Control Plane, validate their configuration and connectivity,
+and then decommission the self-managed Data Plane instances.
+
+See the [Data Plane hosting options](/gateway/topology-hosting-options/) for more information. The easiest
+way to deploy new Data Planes is using the {{site.konnect_short_name}} Gateway Manager, which provides integrated 
+launchers for popular operating systems and compute platforms. 
+
+### APIOps
+
+In {{site.konnect_short_name}}, there are additional options for managing the API deployment lifecycle, as compared to {{site.base_gateway}}:
+
+* **decK**: If you're using [deck](/deck/) to manage your {{site.base_gateway}} configuration, you can continue to use
+the tool to managing your {{site.konnect_short_name}} configuration. The decK CLI 
+[supports {{site.konnect_short_name}} Control Plane configuration](/deck/gateway/konnect-configuration/)
+by providing additional flags that configure the tool to connect to a particular Control Plane using access tokens.
+
+* **Terraform**: Kong provides a Terraform provider for managing a full {{site.konnect_short_name}} deployment, 
+including Control Planes, Control Plane groups, Data Plane configuration, and more. The 
+[{{site.konnect_short_name}} Terraform Provider](/terraform/) can be used independently
+or in conjunction with decK to manage your API deployment lifecycle.
+
+* **{{site.kgo_product_name}}**: The {{site.kgo_product_name}} (KGO) is available for teams 
+that want to use standardized Kubernetes APIs to manage their {{site.konnect_short_name}} deployments.
```

Comment:
How would yo ufeel about removing this section as well. We can add it back if someone requests it.

Reply:
That works for me since it's just info about tools and not concrete migration steps